### ISSUE TITLE
Low Val Accuracy on CIFAR 100

### ISSUE THREAD

**Issue by chaitanyaanimesh** [GuillaumeErhard/Supervised_contrastive_loss_pytorch#2]
Do you mind sharing the parameters you used for training CIFAR 100? For me validation accuracy seems to be saturating at 40% (using resnet-20 though) even with Auto Augment=True. I am interested in knowing the learning rate, batch size, resnet model and temperature you used for CIFAR-100.

**Comment by Angelina1996** [comment]
Could you please guide how to run this code for ImageNet dataset
